Adds filtering/sorting ability on list of bounties.

Christopher-Stevers opened the pull request:
Allows user to sort by time and price, and filter out empty bounties.

What do you think of the chunky toggle on the smaller screen width?
![image](https://user-images.githubusercontent.com/72156679/155263824-073c6bb4-30ea-46d2-b04e-d09aac50fe1f.png)
![image](https://user-images.githubusercontent.com/72156679/155263853-4c319b4b-6e25-4bc4-8c77-6c79b11bfe52.png)

Closes #65

Christopher-Stevers committed:
corrected AboutBody.js to prevent valuses from  closed issues from showing up in the total value locked list

Christopher-Stevers committed:
replaced bounty list in organization route with BountyList component and added filtering and sorting within it. Also changed Toggle inputs from bools to strings.

Christopher-Stevers committed:
merged upstream and tweaked display

Christopher-Stevers committed:
Merge branch 'development' of https://github.com/OpenQdev/OpenQ-Frontend

FlacoJones commented:
For the Oldest/Newest and Highest/Lowest, can we consolidate them into a single dropdown like [this](https://imgur.com/a/tBBa4hg)?

Then for the Funded/Unfunded toggle, maybe just a checkbox that says "Show Unfunded"

Both can be under the Search Issue bar. Toggle on left, level with the dropdown on right.

Christopher-Stevers committed:
added dropdown and checkboxes for filtering bounties

Christopher-Stevers committed:
Merge branch 'development' of https://github.com/OpenQdev/OpenQ-Frontend

Christopher-Stevers committed:
added borders around filter checkboxes and fixed some weird overflow issues

Christopher-Stevers commented:
![image](https://user-images.githubusercontent.com/72156679/155413683-a275a3ee-b0c1-4b1c-b7ba-1a6dc364245e.png)
@FlacoJones Made UI changes as requested, refactored sorting methods and added claim based filtering. If we upgrade tailwind we'll be able to access accent color on the checkboxes.

FlacoJones commented:
Love it. lgtm. merging!
